docker-compose.yml

```diff
@@ -2,6 +2,8 @@ version: '3'
 services:
   server:
     image: app:build
```

Comment:
Not need to specify the image here, docker-compose will build this on its own when needed.

Reply:
Good call I will remove that image field